src/scripts/main.js

```diff
@@ -1,3 +1,14 @@
 'use strict';
 
 // write code here
+const wallElement = document.querySelector('.wall');
+const spiderElement = document.querySelector('.spider');
+
+const currentSpiderWidth = spiderElement.clientWidth / 2;
+const currentSpiderHeight = spiderElement.clientHeight / 2;
+const spiderPositionVertical = wallElement.clientWidth / 2 - currentSpiderWidth;
```

Comment:
The variable `spiderPositionVertical` is actually calculating the horizontal position of the spider. Consider renaming it to `spiderPositionHorizontal` for clarity.